app/internal/wm/os_js.go

```diff
@@ -583,6 +591,26 @@ func (w *window) windowMode(mode WindowMode) {
 	}
 }
 
+func (w *window) navigationColor(c color.NRGBA) {
+	if c.A != 255 {
```

Comment:
Delete this check. The conversion routine should work regardless of the alpha value.